xbmc/windows/GUIWindowFileManager.cpp

```diff
@@ -1279,3 +1289,30 @@ const CFileItem& CGUIWindowFileManager::CurrentDirectory(int indx) const
 {
   return *m_Directory[indx];
 }
+
+void CGUIWindowFileManager::TextView(const std::string& path)
```

Comment:
since this method is static, perhaps we could move it to some utility class?

Reply:
moved to a static.